b2/console_tool.py

```diff
@@ -258,6 +286,24 @@ def test_raw_api(self, args):
 
     # bucket
 
+    def cancel_all_unfinished_large_files(self, args):
```

Comment:
This is logic implemented in `ConsoleTool`. I think it should be done in Bucket and just called from here, like other functions (except sync)